packages/mask/src/plugins/Predict/Azuro/hooks/useFetchUserGames.ts

```diff
@@ -0,0 +1,60 @@
+import { useAsyncRetry } from 'react-use'
+import { BetStatus, ConditionStatus, UserBet, UserBetsRawData, UserFilter } from '../types'
+import { useAccount, useChainId } from '@masknet/plugin-infra/web3'
+import type { AsyncStateRetry } from 'react-use/lib/useAsyncRetry'
+import { NetworkPluginID } from '@masknet/web3-shared-base'
+import { betTypeOdd } from '../helpers/index.js'
+import { fetchUserBets } from '../api/index.js'
+
+export function useFetchUserGames(filter: UserFilter): AsyncStateRetry<UserBet[]> {
+    const account = useAccount()
+    const chainId = useChainId(NetworkPluginID.PLUGIN_EVM)
```

Comment:
Learn more about `ChainContext` and `NetworkContext`: https://docs.mask.io/docs/web3/hooks.

```suggestion
    const { account, chainId } = useChainContext<NetworkPluginID.PLUGIN_EVM>()
```

Reply:
It seems that useChainContext is not exported from any package. Am I right ?